### ISSUE TITLE
osu! considers the old set difficulty rather the new one, even after trying to rename it the same name, it kept being considered the old set, not the new set.

### ISSUE THREAD

**Issue by bowlingfan** [ppy/osu-stable-issues#469]
I'm not even sure why osu! does this where, if I upload the old set, and then try to upload my new set, it keeps considering that one difficulty to the old set, even after uploading it to a new diff, I checked the beatmapsetID and they are correct to the new set, but its still being considered to update to the latest version.

https://streamable.com/pnwllp <-- This is proof
https://imgur.com/lfWaKjW <-- Another proof but with the .osu

https://osu.ppy.sh/beatmapsets/1186864#osu/2473529
**THE CURRENT VERSION THAT IS BEING CONSIDERED WITH YENMASTER'S NORMAL**

https://osu.ppy.sh/beatmapsets/1186851#osu/2473555
**THE NEW SET BUT YENMASTER'S NORMAL IS NOT BEING CONSIDERED TO THE NEW SET, WHENEVER I DOWNLOAD IT, IT KEEPS TELLING ME TO UPDATE IT.**

**Comment by peppy** [comment]
Can you confirm that you're uploading two copies of the same beatmap?

**Comment by bowlingfan** [comment]
For confirmation, no I did not upload two copies, I had gotten the new set from the featured artist(s) section, and the old set was the same, but it was a few months ago, and I tried to copy the same placement and stuff, checked SOME of the .osu information, and it seems okay?

**Comment by bowlingfan** [comment]
I CAN CONFIRM that without yenmaster's normal, it seems to work normally, but this issue should still be kept for issues where if theres a same difficulty on the same song sets that you own, this error will come up.

**Comment by peppy** [comment]
Just to help me investigate this, can you take screenshots of the places you are seeing this issue?

**Comment by bowlingfan** [comment]
https://imgur.com/UXaEyva
^^ new set 

https://imgur.com/Rn6BP5J
^^ old set

**both were selected to yenmaster's normal.**

https://streamable.com/veqeji <-- me doing the same thing shown in recording.

**Comment by peppy** [comment]
Okay, so what I'm gathering is that you're having issues uploading the second version when you also have the first one loaded in your osu! client, but removing the first set before uploading the new one corrects the issue?

**Comment by bowlingfan** [comment]
Yes, if what your meaning is: **"If you do not have the first set uploaded to osu, and only uploaded the new set, then it will consider the new set, the difficulty."**
